service/src/test/java/org/databiosphere/workspacedataservice/service/NoCacheFilterTest.java

```diff
@@ -31,27 +36,29 @@
 @ActiveProfiles(profiles = "mock-sam")
 @SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
 class NoCacheFilterTest extends TestBase {
+  @Autowired private TwdsProperties twdsProperties;
+
   private static final String versionId = "v0.2";
 
   @Autowired private ObjectMapper mapper;
 
   @Autowired private TestRestTemplate restTemplate;
 
+  private UUID instanceId = UUID.randomUUID();
```

Comment:
I'd leave this un-initialized as a hint to developers that some later code overwrites it:
```suggestion
  private UUID instanceId;
```